ci-operator/step-registry/openshift/upgrade/gcp/ovn-rt/openshift-upgrade-gcp-ovn-rt-workflow.metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+	"path": "openshift/upgrade/gcp/ovn-rt/openshift-upgrade-gcp-ovn-rt-workflow.yaml",
+	"owners": {
+		"approvers": [
+			"fromanirh",
+			"cynepco3hahue",
+			"yanirq"
+		]
+	}
+}
```

Comment:
Hmm, it seems the `make update` removes newlines for the `.metadata.json` files, it might be best to leave those so you're not always fighting with the `update` command.

Reply:
I see, yep in this case let's leave it as it is now.